lib/maps/fan-thermostat.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (Mapper, Service, Characteristic) => ({
+  class: ['thermostat', 'heatpump'],
+  mainService: 'thermostat',
+  service: Service.Fanv2,
+  // in the context of a thermostat service, device `onoff` is mapped to thermostat power and not to fan speed
+  onUpdate: ({ service }) => {
+    service.getCharacteristic(Characteristic.Active).updateValue(Characteristic.Active.ACTIVE);
```

Comment:
Always display the fan speed configured on the device, even if the thermostat is off.

We also could map with thermostat_mode and display it only if the thermostat is on.